go.mod

```diff
@@ -30,4 +30,4 @@ require (
 
 go 1.13
 
-replace github.com/lightningnetwork/lnd => github.com/lightningnetwork/lnd v0.12.0-beta
+replace github.com/lightningnetwork/lnd => github.com/lightningnetwork/lnd v0.12.1-beta
```

Comment:
Because the version is now higher, the replace apparently isn't needed anymore. I included this commit in the bugfix PR #186 but removed the replace. So we can close this if you don't mind me taking it over.